utils/generate_types/overrides-test.d.ts

```diff
@@ -361,6 +361,8 @@ export type ExpectMatcherState = {
   isNot: boolean;
   promise: 'rejects' | 'resolves' | '';
   utils: ExpectMatcherUtils;
+  expand: boolean;
```

Comment:
- What is this property?
- Let's manually add JSDocs to properties here? At least for `isNot` and `timeout`.

Reply:
> * What is this property?

Removed.

> * Let's manually add JSDocs to properties here? At least for `isNot` and `timeout`.

Done.